ci/dev/watch.ts

```diff
@@ -1,148 +1,204 @@
-import * as cp from "child_process"
+import { spawn, fork, ChildProcess } from "child_process"
+import del from "del"
+import { promises as fs } from "fs"
 import * as path from "path"
-import { onLine } from "../../src/node/util"
-
-async function main(): Promise<void> {
-  try {
-    const watcher = new Watcher()
-    await watcher.watch()
-  } catch (error: any) {
-    console.error(error.message)
-    process.exit(1)
-  }
+import { CompilationStats, onLine, OnLineCallback, VSCodeCompileStatus } from "../../src/node/util"
+
+interface DevelopmentCompilers {
+  [key: string]: ChildProcess | undefined
```

Comment:
This seems to help non-generic functions like `Object.entries` play nice with the type signature.